<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginInputDialog</class>
 <widget class="QDialog" name="PluginInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="lSheets">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Sheets</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="horizontalLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="lSheetWidth">
               <property name="text">
                <string>Width:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leSheetWidth">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lSheetHeight">
               <property name="text">
                <string>Height:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="leSheetHeight">
               <property name="text">
                <string>3</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="lPlugins">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Plugins</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetPlugins">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnEnable">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDisable">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Disable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnEnableAll">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Enable all</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDisableAll">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Disable all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Shapes</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QListView" name="listViewShapes"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0,0">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnAdd">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemove">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnClear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlMainControls" stretch="0,0,0,0,1,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadSVG">
         <property name="text">
          <string>Load SVG...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRandom">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Random</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lStatus">
         <property name="text">
          <string>Status</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leSheetWidth</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PluginInputDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSheetHeight</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PluginInputDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetPlugins</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>PluginInputDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>PluginInputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetPlugins</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PluginInputDialog</receiver>
   <slot>updateControls()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateControls()</slot>
 </slots>
</ui>
